app/workers/reactions/update_reactable_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Reactions
+  class UpdateReactableWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
```

Comment:
I would raise the priority, what do you think @mstruve ?

Reply:
Hmmm, both of the actions in this worker are simple very fast SQL updates. I'm wondering if we save ourselves the complexity and do them inline without creating a whole job for it. @rhymes thoughts?